tezos/new_context/bin/context_action_file_replayer.rs

```diff
@@ -525,22 +534,36 @@ fn perform_context_action(
             value,
             new_tree_id,
             ..
-        } => (current_context.add(&key, value)?, Some(new_tree_id)),
+        } => {
+            let key: Vec<_> = key.iter().map(String::as_str).collect();
```

Comment:
@sebastiencs this conversion between key representation repeats a lot, can it be abstracted?